src/runtime-rs/crates/hypervisor/src/qemu/inner.rs

```diff
@@ -0,0 +1,134 @@
+use anyhow::Result;
+
+use crate::{HypervisorConfig, VcpuThreadIds};
+use kata_types::capabilities::{Capabilities, CapabilityBits};
+
+unsafe impl Send for QemuInner {}
+unsafe impl Sync for QemuInner {}
+
+pub struct QemuInner {
+    config: HypervisorConfig,
+}
+
+impl QemuInner {
+    pub fn new() -> QemuInner {
+        QemuInner {
+            config: Default::default(),
+        }
+    }
+
+    pub(crate) async fn prepare_vm(&mut self, _id: &str, _netns: Option<String>) -> Result<()> {
+        info!(sl!(), "Preparing QEMU VM");
+        Ok(())
+    }
+
+    pub(crate) async fn start_vm(&mut self, _timeout: i32) -> Result<()> {
+        info!(sl!(), "Starting QEMU VM");
+
+        let mut command = std::process::Command::new(&self.config.path);
+
+        command
+            //.current_dir(bundle_path)
+            //.stdin(std::process::Stdio::null())
+            //.stdout(std::process::Stdio::null())
+            //.stderr(std::process::Stdio::null())
+            .arg("-kernel")
+            .arg(&self.config.boot_info.kernel)
+            .arg("-m")
+            .arg(format!("{}M", &self.config.memory_info.default_memory))
+            .arg("-initrd")
+            .arg(&self.config.boot_info.initrd)
+            .arg("-vga")
+            .arg("none")
+            .arg("-nodefaults")
+            .arg("-nographic");
+
+        println!("qemu cmd: {:?}", command);
```

Comment:
Use `info`/... to replace `println`

Reply:
Right, so I'm keeping this local for now. I'll only be able to use `info!()` once I figure out how to set up logging (to console, not file) in shim-ctl.